src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -656,7 +656,7 @@ void EIRPlantLoopHeatPump::calcPowerUsage(EnergyPlusData &state)
     // compute power usage
     if (this->thermosiphonDisabled(state)) {
         this->powerUsage = (this->loadSideHeatTransfer / this->referenceCOP) * eirModifierFuncPLR * eirModifierFuncTemp *
-                           this->defrostPowerMultiplier * this->cyclingRatio;
+                           this->defrostPowerMultiplier;
```

Comment:
The `cyclingRatio` is already applied when calculating the `loadSideHeatTransfer`:

https://github.com/NREL/EnergyPlus/blob/e5c5f38d7ce6f007763ba80bada6e17a9fba0042/src/EnergyPlus/PlantLoopHeatPumpEIR.cc#L621-L622